sabnzbd/nzbstuff.py

```diff
@@ -893,8 +893,8 @@ def __init__(self, filename, pp, script, nzb=None,
         if not self.password and self.meta.get('password'):
             self.password = self.meta.get('password', [None])[0]
 
-        # Set nzo save-delay to 6 sec per GB with a max of 5 min
-        self.save_timeout = min(6.0 * float(self.bytes) / GIGI, 300.0)
+        # Set nzo save-delay to 30 seconds
+        self.save_timeout = 30
```

Comment:
Before, a 5G job would save every 30 seconds and a 20G job every 2 minutes.
Now it's 30 sec always. This will make 20G jobs slower due to all the saving.